applications/EmpireApplication/test_examples/wrapper_example_field_CoSim/kratos_client_1.py

```diff
@@ -7,6 +7,7 @@
 import os
 import ctypes as ctp
 import numpy as np
+import sys
```

Comment:
why not importing sys ? 
asking because I do not see any statements using this ?!

Reply:
it is already used, the import was missing for some reason ...